documentation/API-design-guidelines.md

```diff
@@ -1298,6 +1299,7 @@ servers:
         default: http://localhost:9091
         description: API root, defined by the service provider, e.g. `api.example.com` or `api.example.com/somepath`
 ```
+If more than one server object instances are listed, each `servers[*].url` property must have the same string for the `api-name` and `api-version` in respective instance.
```

Comment:
@rartych, I don't think I had this sentence formulated correctly in my original suggestion.  I suggest the following as the final string:
If more than one server object instances is listed, the `servers[*].url` property of each instance must have the same values for the `api-name` and `api-version`.

Reply:
Corrected